Misc/NEWS

```diff
@@ -764,6 +764,9 @@ Library
 - Issue #29581: ABCMeta.__new__ now accepts **kwargs, allowing abstract base
   classes to use keyword parameters in __init_subclass__. Patch by Nate Soares.
 
+- Issue #29822: inspect.isabstract() now works during __init_subclass__. Patch
```

Comment:
Move this at the start of the Library section. Use two spaces between sentences.